src/dbus_api/pool.rs

```diff
@@ -127,16 +115,14 @@ fn destroy_filesystems(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
                 dbus_context.actions.borrow_mut().push_remove(op);
             }
 
-            let return_value = uuids
+            let return_value: Vec<String> = uuids
```

Comment:
Why is type ```Vec<String>``` instead of ```Vec<&str>```?

Reply:
uuid.simple() is giving us an owned string.